components/map.tsx

```diff
@@ -0,0 +1,19 @@
+export default function Map() {
+    return (
+        <div className="lg:relative bg-blue-900 pb-4 w-screen m-auto">
```

Comment:
Maybe use `<address>` tag.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

And format it "multline":

> Edeltech Ltd
> Chemin Saint-Hubert 5
> 1950 Sion
> Valais
> Switzerland
>